frontend/src/pages/LandlordPage.tsx

```diff
@@ -73,70 +78,87 @@ const dummyRatingInfo: RatingInfo[] = [
 
 const LandlordPage = (): ReactElement => {
   const { landlordId } = useParams<Record<string, string | undefined>>();
-  const [width, setWidth] = useState(window.innerWidth);
   const [landlordData] = useState(dummyData);
   const [aveRatingInfo] = useState(dummyRatingInfo);
-  const breakpoint = 600;
+
+  const [reviewOpen, setReviewOpen] = useState(false);
```

Comment:
Might want to parametrize these `useState` hooks so that they don't get inferred to just be a falsy value.

Reply:
Which hooks are you talking about? landlordData and aveRatingInfo?

Reply:
Lines 84-85: you can do `const [reviewOpen, setReviewOpen] = useState<boolean>(false);` for example

Reply:
Is this needed? Typescript seems to infer the type if you put "false"
![image](https://user-images.githubusercontent.com/3837473/112534147-1c4c6b80-8d81-11eb-91fe-d2acaab25552.png)

Reply:
ok my bad, it only does this when you supply an empty array as the init value. disregard this